silkworm/silkrpc/core/evm_trace.hpp

```diff
@@ -327,10 +327,24 @@ struct TraceDeployResult {
     std::optional<evmc::address> contract_creator;
 };
 
+struct TraceEntry {
+    std::string type;
+    int32_t depth;
+    evmc::address from;
+    evmc::address to;
+    std::string value;
+    std::string input;
+};
+
+struct TraceEntriesResult {
+    std::vector<TraceEntry> entries;
+};
```

Comment:
Probably you don't need a struct here and you can simplify this to:

`using TraceEntriesResult = std::vector<TraceEntry>;`